src/network/serverpackethandler.cpp

```diff
@@ -180,7 +180,7 @@ void Server::handleCommand_Init(NetworkPacket* pkt)
 	if (player && player->getPeerId() != PEER_ID_INEXISTENT) {
 		actionstream << "Server: Player with name \"" << playername <<
 			"\" tried to connect, but player with same name is already connected" << std::endl;
-			DenyAccess(peer_id, SERVER_ACCESSDENIED_ALREADY_CONNECTED);
```

Comment:
I don't fully understand why you changed this. The previous indentation seems right to me. The `DenyAccess(...)` statement is a child of the `if`, and not the `actionstream << ...`.